### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 0xtdey**
Made a few minor changes here and there, to make it more reader friendly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # ***imgman***
 
-imgman is an image manipulation website built with django, which provides options for converting an image to blur, black and white, grayscale, pdf.
+imgman is a unique web app which is provides myriad of image manipulation options like blurring of image, black and white filters, converting to pdf, adding grayscale eefect. And it's built with django a python framework.
```
Comment (kailashchoudhary11) [inline README.md:3]:
```suggestion
imgman is a unique web app which provides myriad of image manipulation options like blurring of image, black and white filters, converting to pdf, adding grayscale effect, resizing the image. It's built with django, a python web framework.
```

File: README.md [hunk 2]
```diff
@@ -31,17 +31,17 @@ imgman is an image manipulation website built with django, which provides option
 
 ## Background
 
-This project, ***imgman***, is a project initiative founded by <https://github.com/kailashchoudhary11>. It is a platform that allows users to manipulate images.
+This project, ***imgman***, is an initiative by <https://github.com/kailashchoudhary11>. It is a platform that allows users to apply unique filters and manipulate images and export them.
```
Comment (kailashchoudhary11) [inline README.md:34]:
```suggestion
This project, ***imgman***, is an initiative by [Kailash Choudhary](https://github.com/kailashchoudhary11). It is a platform that allows users to apply unique filters and manipulate images and export them.
```

**Review by kailashchoudhary11 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kailashchoudhary11) [reply README.md:3]: Please Resolve these errors @tamoghna-dey

Comment (0xtdey) [reply README.md:3]: Yes sure, do I need to re-edit in the same PR or make another and edit it again?

Comment (kailashchoudhary11) [reply README.md:3]: > Yes sure, do I need to re-edit in the same PR or make another and edit it again?

yes you can also click on the commit suggestion button

**Pull request closed without merging**